lib/Controller/Applications.php

```diff
@@ -115,15 +115,23 @@ public function displayPage(Request $request, Response $response)
     {
         // Load all connectors and output any javascript.
         $connectorJavaScript = [];
+        $partnersArr = [];
```

Comment:
This is specific to the SSP connector and therefore it isn't appropriate to add it to the applications page. We're using this as a source of data for the partners dropdown. Once we have implemented the proxy route above, we can fetch the partners on form load and populate the list, or even use a search query from the select2 itself (if we format our method to return in "grid" format accordingly).

Reply:
Removed changes on this as well, thank you!